client/src/components/History/CurrentHistory/HistoryFilters/HistoryFilters.vue

```diff
@@ -197,6 +197,12 @@ export default {
             this.onToggle();
             this.filterSettings["create_time>"] = this.create_time_gt;
             this.filterSettings["create_time<"] = this.create_time_lt;
+            if (this.filterSettings["deleted:"] == undefined) {
```

Comment:
Can we have a test for this, should it use `===`, and under which circumstances is `this.filterSettings["deleted:"]` undefined ? A comment may help future us here.

Reply:
The `filterSettings` object is cleared when `deleted` or `visible` are set to `any`. This is what causes this bug, unless we explicitly reset those values when a search is performed again. Adding a comment here for that for now will help so will do that.
Other than that, working on another PR where I am trying to create a reusable `FilterMenu` component that will fix these bugs.

Reply:
Sounds good, thank you!